agimus_controller/agimus_controller/robot_model_factory/obstacle_params_parser.py

```diff
@@ -0,0 +1,201 @@
+import yaml
+import numpy as np
+import pinocchio as pin
+from pathlib import Path
+from hppfcl import Sphere, Box, Cylinder, Capsule
+
+
+class ObstacleParamsParser:
```

Comment:
overall this whole parser lacks comments, i can fix it for the next PR

Reply:
https://github.com/agimus-project/agimus_controller/issues/107